PerformanceMetrics/CMakeLists.txt

```diff
@@ -62,6 +63,8 @@ set_target_properties(${MODULE_NAME} PROPERTIES
         CXX_STANDARD 11
         CXX_STANDARD_REQUIRED YES)
 
+target_include_directories(${MODULE_NAME} PRIVATE ../helpers)
```

Comment:
Could you please move this into the "SYSLOG" elseif above? The ../helpers will not be available in all builds (for example ours :) ) and this line will break those builds

Reply:
Yes. I moved to SYSLOG portion.